packages/material-ui/src/SnackbarContent/SnackbarContent.js

```diff
@@ -83,6 +83,16 @@ SnackbarContent.propTypes = {
    * The message to display.
    */
   message: PropTypes.node,
+  /**
+   * The role of the SnackbarContent. If the SnackbarContent
+   * includes interactive controls, such as a link, "alertdialog"
+   * should be used; otherwise use "alert".
+   */
+  role: PropTypes.oneOf(['alertdialog', 'alert']),
+};
+
+SnackbarContent.defaultProps = {
```

Comment:
Notice that we don't use `.defaultProps`, prefer default spread value.

Reply:
Fixed